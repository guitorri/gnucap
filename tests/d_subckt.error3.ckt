
.subckt foo (1 2 3 4)
e1 (1 2 3 4) 1
.ends
v1 (1 0) 1
x1 (1 2) foo
.op
.list
.end
