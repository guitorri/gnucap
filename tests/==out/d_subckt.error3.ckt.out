 
x1: need more nodes
.subckt foo ( 1 2 3 4 )
e1 ( 1 2 3 4 )  1.
.ends foo
v1 ( 1 0 )  DC  1.
x1 ( 1 2 )  foo 
